src/pocketmine/Player.php

```diff
@@ -4140,6 +4140,10 @@ public function removeWindow(Inventory $inventory, bool $force = false){
 
 		if($id !== null){
 			$inventory->close($this);
+			
+			$ev = new InventoryCloseEvent($inventory, $this);
+			$ev->call();
```

Comment:
This will cause InventoryCloseEvent to get called twice if a player closes a container.